force-app/main/educationCloudSettings/lwc/edcReleaseManagementCard/edcReleaseManagementCard.css

```diff
@@ -0,0 +1,30 @@
+.progress-container {
+    height: 30px;
+}
+.progressring-container {
+    max-width: 30px;
+}
+.progressring-content {
+    vertical-align: middle;
+}
+.notification-bell {
+    margin-top: -10px;
+    font-size: 15px;
```

Comment:
Absolute pixel references tend to be something we want to avoid. Is there a relative solution you can use in this case?

Reply:
Replaced with relative units. These styles are needed to absolute position the bell icon